### PR TITLE
Quick fix for incomplete rows on main page

### PR DESCRIPTION
**Pull request by UladBohdan**
screen shot 2018-06-04 at 12 21 26 am
screen shot 2018-06-04 at 12 21 51 am

### CODE REVIEW

**Comment by UladBohdan** [comment]
[do not allow articles to occupy all place in a row when there is less then 4 articles](https://trello.com/c/EvrIldYe/193-do-not-allow-articles-to-occupy-all-place-in-a-row-when-there-is-less-then-4-articles)

**Comment by UladBohdan** [comment]
The changes are live:
http://dev.wir.by

File: components/about/TeamRow.js [hunk 1]
```diff
@@ -16,13 +16,15 @@ const Teammate = ({ image, name, role }) => (
 
 const TeamCol = ({ data }) => (
   <div className="column">
-    <div className="columns">{data.map(user => <Teammate key={user.id} {...user} />)}</div>
+    <div className="columns">
+      {data.map(user => <Teammate key={user.id} {...user} />)}
+      {Array(Math.max(0, 3 - data.length)).fill(<div className="column is-4" />)}
```
Comment (Drapegnik) [inline components/about/TeamRow.js]: please extract page sized to constants (or reuse exist)

**Comment by UladBohdan** [comment]
@Drapegnik could you please fix lint issues coming from Travis? they look weird to me

**Pull request merged**